packages/email-petition/src/city-autocomplete.tsx

```diff
@@ -0,0 +1,172 @@
+"use client";
+
+import { useState, useEffect, useRef } from "react";
+import { Input } from "@dxe/petitions-components/input";
+
+interface CityPrediction {
+  cityAddress: string;
+  cityName: string;
+}
+
+interface CityAutocompleteProps {
+  value: string;
+  onChange: (value: string) => void;
+  onBlur: () => void;
+  disabled?: boolean;
+  placeholder?: string;
+  lat?: number;

```

Comment:
please add a comment explaining what these lat/lng are used for here, and ideally make an interface for lat/lng to avoid primitive obsession / data clump smells.